<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogHMM</class>
 <widget class="QDialog" name="DialogHMM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_HMM">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>48</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Hidden Markov Model</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="hmm_return">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>630</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="hmm_strinput">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>391</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="hmm_strres">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>250</y>
     <width>391</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="hmm_strrun">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QLabel" name="hmm_setupmodel_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Setup Model:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="hmm_setup_trainfile">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>89</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Train File</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="hmm_setup_data">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>89</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hmm_stopword">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stopword:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hmm_stopword_Y">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>185</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hmm_setupmodel_wait">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Training……Please wait.</string>
   </property>
  </widget>
  <widget class="QLabel" name="hmm_testfile_label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Test file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hmm_fileopen">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="hmm_filepath">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>160</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hmm_filetest">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QLabel" name="hmm_filepath_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>File Path:</string>
   </property>
  </widget>
  <widget class="QLabel" name="hmm_fileres">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>200</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hmm_fileres_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>210</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Output:</string>
   </property>
  </widget>
  <zorder>hmm_filepath_label</zorder>
  <zorder>hmm_setupmodel_label</zorder>
  <zorder>label_HMM</zorder>
  <zorder>hmm_return</zorder>
  <zorder>hmm_strinput</zorder>
  <zorder>hmm_strres</zorder>
  <zorder>hmm_strrun</zorder>
  <zorder>hmm_setup_trainfile</zorder>
  <zorder>hmm_setup_data</zorder>
  <zorder>hmm_stopword</zorder>
  <zorder>hmm_stopword_Y</zorder>
  <zorder>hmm_setupmodel_wait</zorder>
  <zorder>hmm_testfile_label</zorder>
  <zorder>hmm_fileopen</zorder>
  <zorder>hmm_filepath</zorder>
  <zorder>hmm_filetest</zorder>
  <zorder>hmm_fileres</zorder>
  <zorder>hmm_fileres_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
